client/overview/modal/progressive-onboarding-eligibility/index.tsx

```diff
@@ -59,75 +60,74 @@ const ProgressiveOnboardingEligibilityModal: React.FC = () => {
 		setModalVisible( false );
 	};
 
-	// Workaround to remove Modal header from the modal until `hideHeader` prop can be used.
-	useEffect( () => {
```

Comment:
Removing the useEffect triggered linting error `'useEffect' is defined but never used`.